src/status_im2/contexts/wallet/events.cljs

```diff
@@ -384,3 +377,127 @@
+ (fn [{:keys [db]} [token stack-id]]
+   {:db (assoc-in db [:wallet :ui :send :token] token)
+    :fx [[:navigate-to-within-stack [:wallet-send-input-amount stack-id]]]}))
+
+(rf/reg-event-fx :wallet/send-select-amount
+ (fn [{:keys [db]} [amount stack-id]]
+   {:db (assoc-in db [:wallet :ui :send :amount] amount)
+    :fx [[:navigate-to-within-stack [:wallet-transaction-confirmation stack-id]]]}))
+
+(rf/reg-event-fx :wallet/suggested-routes-success
+ (fn [{:keys [db]} [suggested-routes timestamp]]
+   (when (= (get-in db [:wallet :ui :send :suggested-routes-call-timestamp]) timestamp)
+     {:db (-> db
+              (assoc-in [:wallet :ui :send :suggested-routes] suggested-routes)
+              (assoc-in [:wallet :ui :send :route] (first (:Best suggested-routes)))
+              (assoc-in [:wallet :ui :send :loading-suggested-routes?] false))})))
+
+(rf/reg-event-fx :wallet/suggested-routes-error
+ (fn [{:keys [db]} [_error]]
+   {:db (-> db
+            (update-in [:wallet :ui :send] dissoc :suggested-routes)
+            (update-in [:wallet :ui :send] dissoc :route)
+            (assoc-in [:wallet :ui :send :loading-suggested-routes?] false))}))
+
+(rf/reg-event-fx :wallet/clean-suggested-routes
+ (fn [{:keys [db]}]
+   {:db (-> db
+            (update-in [:wallet :ui :send] dissoc :suggested-routes)
+            (update-in [:wallet :ui :send] dissoc :route)
+            (update-in [:wallet :ui :send] dissoc :loading-suggested-routes?))}))
+
+(rf/reg-event-fx :wallet/get-suggested-routes
+ (fn [{:keys [db]} [amount]]
+   (let [wallet-address      (get-in db [:wallet :current-viewing-account-address])
+         token               (get-in db [:wallet :ui :send :token])
+         to-address          (get-in db [:wallet :ui :send :to-address])
+         token-decimal       18
+         token-id            (:symbol token)
+         value_              (js/parseFloat amount)
+         network-preferences [1]
+         gas-rates           0 ;low
+         amount-in           (money/mul (money/bignumber value_)
+                                        (money/from-decimal token-decimal))
+         from-address        wallet-address
+         request-params      [0
+                              from-address
+                              to-address
+                              (money/to-hex amount-in)
+                              token-id
+                              []
+                              []
+                              network-preferences
+                              gas-rates
+                              {}]
+         timestamp           (datetime/timestamp)]
+     (prn "=====" request-params)
+     {:db            (-> db
+                         (assoc-in [:wallet :ui :send :loading-suggested-routes?] true)
+                         (assoc-in [:wallet :ui :send :suggested-routes-call-timestamp]
+                                   timestamp))
+      :json-rpc/call [{:method     "wallet_getSuggestedRoutes"
```

Comment:
perhaps to keep this pr small we can make it first just adding the functionality to get the routes? and then add the follow up to also send the transaction? it should unblock more work on both pages then 👍